src/modules/editTask.js

```diff
@@ -0,0 +1,17 @@
+import getLocalStorage from './getStorage.js';
+import setLocalStorage from './setStorage.js';
+
+const editTask = (id) => {
```

Comment:
- [ ] Editing a task overrides the previous task, and task 2 rather gets edited in the local storage. Kindly ensure that the appropriate task edited reflects on the both the frontend and the in the corresponding local storage

**Before Editing First task `one`:**
![image](https://user-images.githubusercontent.com/12745474/185769419-eb59ad7a-3bf0-4253-8d7d-6257ddee7569.png)


**After Editing First take `one`:**
![image](https://user-images.githubusercontent.com/12745474/185769457-94ec91ac-f4e3-4320-ba85-6ad8c2b77c75.png)